### ISSUE TITLE
GPU Hang on vcs ring when encoding h.264 odd resolution clip

### ISSUE THREAD

**Issue by chivakker** [01org/intel-vaapi-driver#318]
Try to encode whatever clip with this command line

yamitranscode -i ./input_16x1088.yuv -W 16 -H 1088 -o ./output_1088.h264 -b 11955 -f 3 -c AVC -s I420 -rcmode CBR

Tried on KBL libva/intel-vaapi-driver ToT master

**Comment by QuPengfei** [comment]
@chivakker would you share the yuv file ? i will try it.

**Comment by QuPengfei** [comment]
@chivakker i create a file in my side and it can be reproduced.

I also try the 32x1080 and it work well.

**Comment by chivakker** [comment]
Yeah 32x1088 works okay, it is just 16x1088. Let me know if you need more information.

**Comment by QuPengfei** [comment]
Just confirm the encoder kernel allow the minimum width/height 32. the 16 is not supported.
